CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ straightforward as possible.
 ### Fixed
 - mobile upload on Android
 - device log. new format support [#565]
+- Log in by key uploading UI issue
```

Comment:
Please specify github issue number in []
And add it to PR header as well

Reply:
done